<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdDialog</class>
 <widget class="QDialog" name="CmdDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>358</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Команды и уставки</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Выбор команды</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" rowspan="2" colspan="4">
       <widget class="QComboBox" name="comboBox_selectCMD"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>IOA</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox_timestamp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>С меткой времени</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" rowspan="2">
       <widget class="QPushButton" name="pushButton_listCMD">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QComboBox" name="comboBox_type">
        <item>
         <property name="text">
          <string>Одноэлементная команда (45, 58)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Двухэлементная команда (46, 59)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Команда пошагового регулирования (47, 60)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Команда уставки, нормализованное значение (48, 61)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Команда уставки, масштабированное значение (49, 62)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Команда уставки, короткое число с плавающей точкой (50, 63)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Строка из 32 битов (51, 64)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_ioa">
        <property name="maximum">
         <number>16777215</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>comboBox_type</zorder>
     <zorder>comboBox_selectCMD</zorder>
     <zorder>pushButton_listCMD</zorder>
     <zorder>label</zorder>
     <zorder>spinBox_ioa</zorder>
     <zorder>checkBox_timestamp</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_setpoint">
     <property name="title">
      <string>Параметры уставки</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Значение</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_value">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_setcommand">
     <property name="title">
      <string>Параметры команды</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>29</y>
        <width>436</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="comboBox_cmdValue">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>ОТКЛ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ВКЛ</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_SCType">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Короткий импульс</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Длинный импульс</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Постоянный выход</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_SE">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Исполнение</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Выбор</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Активировать</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_deactivate">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Деактивировать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
